lambda/health_package/generate_metric_alarms.py

```diff
@@ -330,8 +330,6 @@ def main():
                 "Statistic": "Maximum",
                 "Multiplier": 1.1,
                 "Minimum": 3,
-                # Any lambda running for less than 3 secs should be fine
-                "Maximum": 60,
                 # Any lambda running longer than 1 minute probably needs breaking up
```

Comment:
I think you've removed the wrong comment. The 3 seconds was about the minimum threshold value that's still there.  I think we should add a comment too to say that the maximum is calculated based on the lambda's timeout.